api/v1/storageclusterpeer_types.go

```diff
@@ -25,6 +25,21 @@ import (
 
 type StorageClusterPeerState string
 
+const (
+	// StorageClusterPeerInitializing represents Initializing state of StorageClusterPeerState
+	StorageClusterPeerInitializing StorageClusterPeerState = "Initializing"
+	// StorageClusterPeerOnboarding represents Onboarding state of storageClusterPeer
+	StorageClusterPeerOnboarding StorageClusterPeerState = "Onboarding"
+	// StorageClusterPeerConnected represents Onboarding state of storageClusterPeer
+	StorageClusterPeerConnected StorageClusterPeerState = "Connected"
+	// StorageClusterPeerOffboarding represents Onboarding state of storageClusterPeer
+	StorageClusterPeerOffboarding StorageClusterPeerState = "Offboarding"
+	// StorageClusterPeerFailed represents Failed state of StorageClusterPeerState
+	StorageClusterPeerFailed StorageClusterPeerState = "Failed"
+	// StorageClusterPeerDeleting represents Deleting state of StorageClusterPeerState
+	StorageClusterPeerDeleting StorageClusterPeerState = "Deleting"
```

Comment:
let's use only the things that we are using, in client-op as well we weren't using many and they just add up needing to look what every state entails.

I'm still confused about phase vs state vs status vs conditions, probably this might be a good time to discuss on this (team meeting maybe) and follow a single approach?

Reply:
Updated the status to have `RemoteState` which will denote the onboarding State for the storageClusterPeer.
We can add BlockPoolMirroringState to denote the state of BlockPoolMirroring, in the future as we add more operations for cross odf communication, we could add more states to denote the state of those operations

Reply:
I think my comment is misunderstood, I mean, I see a lot of States here are all these states being used in the code? If not, let's have only what we are using.

Reply:
I have updated the state to be more meaningful and reflect what they are depicting